service_configuration_lib/__init__.py

```diff
@@ -87,9 +87,16 @@ def read_dependencies(dependencies_file):
     return read_yaml_file(dependencies_file)
 
 
-def read_yaml_file(file_name):
+def read_yaml_file(file_name, deepcopy=True):
+    """
+        Reads the given yaml file
```

Comment:
Absolutely thrilling docstring, not saying that docstrings are bad but there are probably more wirthy subjects

Reply:
lol. I think I was gonna add more here but got distracted.